src/main/java/eu/socialsensor/graphdatabases/Neo4jGraphDatabase.java

```diff
@@ -188,17 +187,18 @@ public void shutdownMassiveGraph()
     @Override
     public void shortestPath(Node n1, Integer i)
     {
+        PathExpander pathExpander = PathExpanderBuilder.allTypesAndDirections().add(Neo4jGraphDatabase.RelTypes.SIMILAR,Direction.BOTH).build();
         PathFinder<Path> finder
-            = GraphAlgoFactory.shortestPath(Traversal.expanderForTypes(Neo4jGraphDatabase.RelTypes.SIMILAR), 5);
+            = GraphAlgoFactory.shortestPath(pathExpander, 5);
         Node n2 = getVertex(i);
         Path path = finder.findSinglePath(n1, n2);
 
     }
 
     //TODO can unforced option be pulled into configuration?
     private Transaction beginUnforcedTransaction() {
-        final TransactionBuilder builder = ((GraphDatabaseAPI) neo4jGraph).tx().unforced();
-        return builder.begin();
+        return neo4jGraph.beginTx();
```

Comment:
This is not an unforced transaction. I didn't have the time to play with newest Neo4j and I don't know I this concept exists. If not please the method name to be consistent.